deps/thpool/thpool.c

```diff
@@ -59,6 +59,7 @@ typedef struct job {
 typedef struct {
   job *job;
   bool is_admin;
+  bool is_privileged;
```

Comment:
```suggestion
  bool reserved_for_high_priority;
```
or something like that